dapps/react-dapp-auth/pages/_app.tsx

```diff
@@ -1,27 +1,70 @@
 import "../styles/globals.css";
 import type { AppProps } from "next/app";
-import { ChakraProvider, Container, Flex, Image, Text } from "@chakra-ui/react";
+import {
+  Box,
+  ChakraProvider,
+  Container,
+  Flex,
+  Grid,
+  GridItem,
+  Image,
+  Text,
+} from "@chakra-ui/react";
+import ThemeSwitcher from "../components/ThemeSwitcher";
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
     <ChakraProvider>
-      <Container height="100vh" width="100vw">
-        <Flex py={5} gap={20} height={"100%"} direction="column">
-          <Flex
-            alignItems="center"
-            alignContent="center"
-            justifyContent="space-between"
-            p={5}
-            backgroundColor={"rgba(39,42,42,.95)"}
-            borderRadius={"16px"}
-          >
-            <Image width={10} src="/walletconnect.png" />
-            <Flex gap={5}>
-              <Text fontWeight={900}>Request Authentication</Text>
+      <Container
+        style={{ minWidth: "100vw", height: "100vh" }}
+        className="bg-primary"
+      >
+        <Grid
+          templateAreas={`
+					"header"
+					"main"
+					"footer"
+				`}
+          style={{ height: "100%", width: "100%" }}
+          gridTemplateRows={"50px 3f 20px"}
+          gridTemplateColumns={"1fr"}
+          paddingY="2em"
+        >
+          <GridItem area={"header"}>
+            <Flex
+              alignItems="center"
+              justifyContent="center"
+              gap="5"
+              fontSize={"1.25em"}
+            >
+              <div>Example App</div>
+              <Flex
+                padding="0.5em"
+                borderRadius="16px"
+                className="bg-secondary"
+                gap="2"
+              >
+                <Image
+                  width="1.5em"
+                  height="1.5em"
+                  src="/wc-bg.png"
+                  alt="WC"
+                ></Image>
+                <span>V0.1.6</span>
```

Comment:
Auth version is out of date, we can set this dynamically by importing the package.json for `auth-client` here:

```js
import {version} from '@walletconnect/auth-client/package.json'

```

Reply:
e.g. https://github.com/WalletConnect/web-examples/pull/55/files#diff-6ce896400b5b04cd2c117d6f917d3272dd9996ec2ac86aa4b9255b4cb546b6e9R3